### PR TITLE
Add a default text for Level

### PR DESCRIPTION
**Pull request by frinkfromthedrink**
In case users don't fill out the settings modal, this applies level instead of nothing before the current rank

### CODE REVIEW

File: js/src/forum/components/LevelBar.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default class LevelBar extends Component {
             expPercent = (100 / 135) * ((expTotal) - (expLevel * 135));
 
         return (
-            <div class="PostUser-level" title={expTotal + ' EXP'}>
+            <div class="PostUser-level" title={`${expTotal} ${app.translator.trans('reflar-level-ranks.admin.settings.levelText')}`}>
```
Comment (dsevillamartin) [inline js/src/forum/components/LevelBar.js]: Why is this code repeated? Just use `pointsText`.

**Review by dsevillamartin — CHANGES_REQUESTED** [review]

File: js/src/forum/components/LevelBar.js (hunk 1 above)
Comment (frinkfromthedrink) [reply js/src/forum/components/LevelBar.js]: Where are you getting that from? On my end, it is using app.translator.trans

Comment (frinkfromthedrink) [reply js/src/forum/components/LevelBar.js]: That's for EXP, pointsText returns the default Level, therefore changing EXP with Level

Comment (dsevillamartin) [reply js/src/forum/components/LevelBar.js]: I know it's for EXP. You are still using the translator text for the "level" though... you are not making sense.

**Pull request merged**